README.md

```diff
@@ -72,6 +72,19 @@ Agent Spawn Configuration:
+| Show Environment Tags                        | Bool   | If enabled draws the environment tags as Gizmos                                                                                                        (Can only be seen inside Unity Editor with Gizmos enabled)                                                                                                                                                                                                                                                   |
+| Should Write CSV Results                     | Bool   | If enabled statistics are written to CSV files whenever a simulation is finishes (Not when  the simulator is closed manually)                                                                                                                                                                                                                                                                                                                                       |
+| Statistics Output Path                       | string | The directory where the CSV results are saved upon completing a simulation. Remember to end string with Path.DirectorySeparatorChar                                                                                                                                                                                                                                                                                                                                 |
+| Ticks Per Stats Snapshot                     | Int    | The frequency of data collection from the simulation. If this value is 1 the state of the simulation is measured and saved at every logic tick                                                                                                                                                                                                                                                                                                                      |
+| Populate Adjacency And Com Groups Every Tick | Bool   | The adjacency matrix used inside the [CommunicationManager.cs](Assets/Scripts/Robot/CommunicationManager.cs) is populated lazily. Enabling this setting will make it eager. This can be useful for gathering statistics regarding communication ranges to test, if agents are at any time outside communication range, as opposed to testing only when communication actually occurs. Enabling this does, however, decrease performance, sometimes significantly so |
```

Comment:
```suggestion
| Populate Adjacency And Com Groups Every Tick | Bool   | The adjacency matrix used inside the [CommunicationManager.cs](Assets/Scripts/Robot/CommunicationManager.cs) is populated lazily. Enabling this setting will make it eager. This can be useful for gathering statistics regarding communication ranges to test if agents are at any time outside communication range, as opposed to testing only when communication actually occurs. Enabling this does, however, decrease performance - sometimes significantly so |
```